packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -10,30 +10,24 @@ import { isFluidHandle } from "@fluidframework/runtime-utils/internal";
 import {
```

Comment:
This file and its functions have not been renamed yet to help simplify the diff.
A follow-up change will rename them.